Tests/StoreKitUnitTests/StoreKit2/StoreKit2TransactionFetcherTests.swift

```diff
@@ -148,6 +148,78 @@ class StoreKit2TransactionFetcherTests: StoreKitConfigTestCase {
         expect(result) == transaction
     }
 
+    // MARK: - receipt
+
+    func testGeneratesReceipt() async throws {
+        _ = try await self.createTransaction(productID: Self.product1,
+                                             finished: false)
+        _ = try await self.createTransaction(productID: Self.product2,
+                                             finished: false)
+        let receipt = await self.fetcher.receipt
+        expect(receipt.transactions.count) ==  2
+        expect(receipt.subscriptionStatus.count) == 2
+        expect(receipt.environment) == .xcode
+
+        if #available(iOS 16.0, macOS 13.0, tvOS 16.0, watchOS 9.0, *) {
+            expect(receipt.bundleId) == Bundle.main.bundleIdentifier
+            expect(receipt.originalApplicationVersion).notTo(beEmpty())
+            expect(receipt.originalPurchaseDate).notTo(beNil())
+        } else {
+            // AppTransaction is not available on iOS 15
+            expect(receipt.bundleId).to(beEmpty())
+            expect(receipt.originalApplicationVersion).to(beNil())
+            expect(receipt.originalPurchaseDate).to(beNil())
+
+            self.logger.verifyMessageWasLogged(
+                Strings.storeKit.sk2_app_transaction_unavailable,
+                level: .warn
+            )
+        }
+    }
+
+    // MARK: - unverified transactions
+
+    func testHandlesUnverifiedTransactions() async throws {
```

Comment:
Can you rename these to include the method that is being tested?
I'd prefix these with `testVerifiedTransaction...`